tests/test_transformDiaSourceCatalog.py

```diff
@@ -182,21 +182,25 @@ def test_bit_unpacker(self):
             else:
                 self.assertFalse(flag['base_PixelFlags_flag_offimage'])
 
+    def test_flag_existence_check(self):
+        unpacker = UnpackApdbFlags(self.config.flagMap, "DiaSource")
+
+        self.assertTrue(unpacker.flagExists('base_PixelFlags_flag'))
+        self.assertFalse(unpacker.flagExists(''))
+        with self.assertRaises(ValueError):
+            unpacker.flagExists(['base_PixelFlags_flag'], columnName='doesNotExist')
+
     def test_flag_bitmask(self):
         """Test that we get the expected bitmask back from supplied flag names.
         """
-
         unpacker = UnpackApdbFlags(self.config.flagMap, "DiaSource")
 
-        self.assertEqual(unpacker.makeFlagBitMask(['']),
-                         np.uint64(0))
-        self.assertEqual(unpacker.makeFlagBitMask(['base_PixelFlags_flag'],
-                         columnName='doesNotExist'),
-                         np.uint64(0))
-        self.assertEqual(unpacker.makeFlagBitMask(['base_PixelFlags_flag']),
-                         np.uint64(1))
-        self.assertEqual(unpacker.makeFlagBitMask(['base_PixelFlags_flag_offimage']),
-                         np.uint64(2))
+        with self.assertRaises(ValueError):
+            unpacker.makeFlagBitMask([''])
+        with self.assertRaises(ValueError):
+            unpacker.makeFlagBitMask(['base_PixelFlags_flag'], columnName='doesNotExist')
```

Comment:
I'd suggest both of these use `assertRaisesRegex` with some of the expected text.